tests/data/README.md

```diff
@@ -175,3 +175,31 @@ License: [same as libavif](https://github.com/AOMediaCodec/libavif/blob/main/LIC
 
 Source: Personal photo converted with `avifenc --grid 1x5 --yuv 420` at
 commit [632d131](https://github.com/AOMediaCodec/libavif/commit/632d13188f9b7faa40f20d870e792174b8b5b8e6).
+
+## From Github Issues
```

Comment:
Nit/optional: We can also just use the existing section "Grid" (at line 168) and not start a new section.

Reply:
Done.